testsuites/syncgateway/functional/tests/test_db_online_offline.py

```diff
@@ -873,6 +737,178 @@ def test_multiple_dbs_unique_buckets_lose_tap(params_from_base_test_setup, sg_co
             assert error_tuple[1] == 503
 
 
+def rest_scan(sync_gateway, db, online, num_docs, user_name, channels):
```

Comment:
Nope, nothing has changed.  I moved the `rest_scan` function to the bottom of the file since it was confusing to see it at the top... when I was expecting to see actual test cases.